lib/exercises.rb

```diff
@@ -2,19 +2,63 @@
 
 # This method will return an array of arrays.
 # Each subarray will have strings which are anagrams of each other
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: O(n log n) ?? because of the word sort section
+# Space Complexity: O(n) where n is the length of the string array
 
 def grouped_anagrams(strings)
```

Comment:
If the words can be any length the time complexity is O(n * m log m) where m is the length of the words since you're sorting each word.